js/ui/list.d.ts

```diff
@@ -54,77 +54,80 @@ export interface ScrollInfo {
 }
 
 /** @public */
-export type ContentReadyEvent = EventInfo<dxList>;
+export type ContentReadyEvent<TItem extends string | Item<any> | any = any, TKey = any> = EventInfo<dxList<TItem, TKey>>;
 
 /** @public */
-export type DisposingEvent = EventInfo<dxList>;
+export type DisposingEvent<TItem extends string | Item<any> | any = any, TKey = any> = EventInfo<dxList<TItem, TKey>>;
 
 /** @public */
-export type GroupRenderedEvent = EventInfo<dxList> & {
+export type GroupRenderedEvent<TItem extends string | Item<any> | any = any, TKey = any> = EventInfo<dxList<TItem, TKey>> & {
     readonly groupData?: any;
     readonly groupElement?: DxElement;
     readonly groupIndex?: number;
 };
 
 /** @public */
-export type InitializedEvent = InitializedEventInfo<dxList>;
+export type InitializedEvent<TItem extends string | Item<any> | any = any, TKey = any> = InitializedEventInfo<dxList<TItem, TKey>>;
```

Comment:
We could extract local type:
```ts
type ItemLike = string | Item<any> | any;
```

```suggestion
export type InitializedEvent<TItem extends ItemLike = any, TKey = any> = InitializedEventInfo<dxList<TItem, TKey>>;
```